tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/views/timeline/timeline_component.ts

```diff
@@ -34,7 +34,7 @@ export interface ExecutionDigestForDisplay {
   standalone: false,
   selector: 'timeline-component',
   templateUrl: './timeline_component.ng.html',
-  styleUrls: ['./timeline_component.css'],
+  styleUrls: ['./timeline_component.scss'],
```

Comment:
This doesn't look right. Typically we compile the scss to css then import the compiled file
```suggestion
  styleUrls: ['./timeline_component.css'],
```

Reply:
SG - I thought I had renamed the file but it looks like it wasnt